tardis/model.py

```diff
@@ -6,8 +6,9 @@
 import numpy as np
 import pandas as pd
 from astropy import constants, units as u
-from astropy.utils import deprecated
+from astropy.utils.decorators import deprecated
 
+from tardis.io.util import to_hdf
```

Comment:
I feeld it would look cleaner if we did `import tardis.io.util` and then use `tardis.io.util.to_hdf`.
Just a remark, not something need to fix before merging.